packages/core/src/components/menu/menuSection.tsx

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I don't think this component is necessary. It makes the API for defining MenuItems look different from the current API. This syntax change from JSX props -> JS object key-value pairs feels a bit weird and I don't want to make users refactor their code like this:

before:

```tsx
<MenuDivider title="Edit" />
<MenuItem icon="cut" text="Cut" label="⌘X" />
<MenuItem icon="duplicate" text="Copy" label="⌘C" />
<MenuItem icon="clipboard" text="Paste" label="⌘V" disabled={true} />
<MenuDivider title="Text" />
```

after:

```tsx
<MenuSection
    sectionTitle="Edit"
    items={[
        {
            icon: "cut",
            id: "cut",
            label: "⌘X",
            text: "Cut",
        },
        {
            icon: "duplicate",
            id: "duplicate",
            label: "⌘C",
            text: "Copy",
        },
        {
            disabled: true,
            icon: "clipboard",
            id: "clipboard",
            label: "⌘V",
            text: "Paste",
        },
    ]}
/>
```